### ISSUE TITLE
Share action indicator position fix

### ISSUE THREAD

**Issue by vvorski** [Vorski-Imagineering/share#51]
- move share sent indicator closer to image
- move share confirmed indicator closer to image
- make share confirmed indicator green
- make speach bouble colour same as share-type icon colour

![screen shot 2013-11-16 at 01 05 42](https://f.cloud.github.com/assets/1789426/1651566/70940628-5ae2-11e3-9507-169ada3286b8.png)

**Comment by vvorski** [comment]
@konewa gdzie to staneło?

**Comment by vvorski** [comment]
- design requirement agreed.

**Comment by vvorski** [comment]
See proposal in #151

**Comment by rimbambala** [comment]
![share-06d](https://cloud.githubusercontent.com/assets/6513797/2568772/fcd8371a-b8e5-11e3-9e5f-fc724b41f446.jpg)

**Comment by vvorski** [comment]
@rimbambala yes but I think will look better with just a checkmark, without the green background - similar to #34

**Comment by rimbambala** [comment]
@vvorski  Just a check mark? Without any border? I wouldn't like it to look as a checkbox. In this case I would change the text bobble in the same manner.

**Comment by vvorski** [comment]
@rimbambala yes, of course change both. If checkmark is unclear some other icon? thumbs up? but I think just drawn without coloured background is lighter and matches the overall UI better. 

Thank you!

**Comment by rimbambala** [comment]
I was thinking not to use bounding box around the check mark. This is how it looks:
![share-06e](https://cloud.githubusercontent.com/assets/6513797/2578185/d0493824-b98b-11e3-8f96-59e922f4ed13.jpg)

**Comment by vvorski** [comment]
Works for me, let's implement it.

**Comment by vvorski** [comment]
Final icons in #54 & #159

**Comment by vvorski** [comment]
PSD in:
https://docs.google.com/a/vorski.com/file/d/0B1nXDt76rlegVTE5cEtjTml0ZFk/edit

share-06.psd

**Comment by vvorski** [comment]
@ZakGhouze please take a look, check, tell me what you think, the new version is at:

http://share-06-test.parseapp.com/#/
